test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestCatalog.java

```diff
@@ -14,19 +14,6 @@
  **/
 package ddf.catalog.test;
 
-import org.junit.After;
```

Comment:
After fixing the compile issue I noticed that some of the imports weren't needed (getting warnings in Eclipse) so I cleaned them up. And that's actually why I had two separate commits initially, one for the pom change and one for the import clean up. Do you want me to undo the import cleanup altogether or create a separate commit for it?